Childrens-Social-Care-CPD/Models/ContextModel.cs

```diff
@@ -2,6 +2,11 @@
 
 namespace Childrens_Social_Care_CPD.Models;
 
+public record PublishDates (
+    DateTime ?FirstPublishedAt,
```

Comment:
Going to assume that only the Date is used from this date DataTime as UTC does not seem. to handled?

Reply:
It is only the date that gets displayed to the user.